src/aro/toolchains/Msvc.zig

```diff
@@ -0,0 +1,41 @@
+const std = @import("std");
+const mem = std.mem;
+
+const system_defaults = @import("system_defaults");
+
+const Compilation = @import("../Compilation.zig");
+const Driver = @import("../Driver.zig");
+const target_util = @import("../target.zig");
+const Toolchain = @import("../Toolchain.zig");
+
+const Msvc = @This();
+
+pub fn defineSystemIncludes(tc: *const Toolchain) !void {
+    if (tc.driver.nostdinc) return;
+
+    const comp = tc.driver.comp;
+
+    if (!tc.driver.nobuiltininc) {
+        try comp.addBuiltinIncludeDir(tc.driver.aro_name);
+    }
+
+    // TODO -imsvc
+    // TODO /external:env:
+    // TODO DIA SDK
+
+
+    if (tc.driver.nostdlibinc) return;
+
+    if (try addSystemIncludeFromEnv(comp, "INCLUDE") or
+        try addSystemIncludeFromEnv(comp, "EXTERNAL_INCLUDE")) return;
+}
+
+fn addSystemIncludeFromEnv(comp: *Compilation, env: []const u8) !bool {
+    const value = std.process.getEnvVarOwned(comp.gpa, env) catch return false;
```

Comment:
There's an `environment` field in Compilation that I created to hold all the environment variables that might be used (to remove the dependency on the `process` api outside of the top-level code that is initializing aro)